config/environment.js

```diff
@@ -77,6 +77,15 @@ module.exports = function(environment) {
     ifa: {
       enabled : false,
       inline  : false
+    },
+
+    torii: {
+      providers: {
+        'stripe-connect': {
+          clientId    : process.env.STRIPE_CLIENT_ID,
```

Comment:
This client_id is supposed to be fetched from the settings endpoint :)

Reply:
@srv-twry I tried that but could not get idea about how to do it, since this file resides outside app and stripe picks its settings from this file. @niranjan94 Please help me if there is a way to do this.

Reply:
@pradeepgangwar you will need to extend the stripe service and customize it to make it work as per our needs.

Reply:
Extend the `stripe-connect` provider. Define the `clientId` property as an alias for the settings client ID property

Reply:
https://github.com/Vestorly/torii/blob/9369efa96eeaf1a5404bab9994ed7c8f805edb1e/addon/providers/oauth2-code.js#L74